apps/api/src/app/portfolio/portfolio-calculator.ts

```diff
@@ -203,11 +203,8 @@ export class PortfolioCalculator {
     const dataGatheringItems: IDataGatheringItem[] = [];
     const firstIndex = transactionPointsBeforeEndDate.length;
 
-    let day = start;
-
-    while (isBefore(day, end)) {
+    for (let day = start; isBefore(day, end); day = addDays(day, step)) {
```

Comment:
Can you use [eachDayOfInterval](https://date-fns.org/v2.29.3/docs/eachDayOfInterval)? I think that's even simpler.

Reply:
I actually looked through the docs for exactly that function, but assumed it would be under "Day" functions. Have replaced it now :+1: